view/admin/utils/getFormData.js

```diff
@@ -80,9 +85,19 @@ function fromTextInput(inputs, jsonData, alterDataFunc) {
  * @returns {Object} with keys to particular DOM elements
  */
 function getInputsAndSelects(form) {
```

Comment:
There are ways but they are verbose to write and maintain. The crux of the method is node.contains, which is well-documented https://developer.mozilla.org/en-US/docs/Web/API/Node/contains